docs/_articles/en/v3.x/salesforce-graph-engine/rules.md

```diff
@@ -71,14 +72,6 @@ The `stripInaccessible` warning is thrown for all `stripInaccessible` checks on
 
 Graph Engine ran into an error while walking this path. Manually verify that you have a sanitizer on the path, and add an engine directive to skip the path. Next, create a GitHub issue for the Code Analyzer team that includes the error and stack trace so we can research and resolve it. After we determine a fix for the issue, check Code Analyzer [Release Information](./en/v3.x/release-information/) for more info.
 
-#### See Also
```

Comment:
On the one hand, it makes sense for "See Also" to be at the bottom. On the other hand, it feels weird to me for the "See Also" links related to this rule to be so far away from the rest of the docs for the rule. Is there a way we can thread that needle?

Reply:
+1